inference-engine/tests/ngraph_functions/src/roi_pooling.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+//
+
+#include <vector>
+#include <memory>
+
+#include "ngraph_functions/builders.hpp"
+
+namespace ngraph {
+namespace builder {
+
+std::shared_ptr<Node> makeROIPooling(const Output<Node>& input,
+                                     const Output<Node>& coords,
+                                     const Shape& output_size,
+                                     const float spatial_scale,
+                                     const ngraph::helpers::ROIPoolingTypes& roi_pool_type) {
+    std::string roi_pool_method = (roi_pool_type == helpers::ROIPoolingTypes::ROI_MAX) ? "max" : "bilinear";
```

Comment:
one more ternary here with `"undef"` could help to signal about the incomplete code changes.

Reply:
Add runtime error by analog with Eltwise builder